src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java

```diff
@@ -67,22 +67,42 @@ public void run() {
 
 		// Assign a label to our nodes
 		nodeLabel = DynamicLabel.label(PropertyTypes.NODELABEL.name());
+		sourceLabel = DynamicLabel.label(PropertyTypes.SOURCE.name());
 		// Recreate our indices
-		execute(new IndexCommand(nodeLabel));
+		execute(new IndexCommand(nodeLabel, sourceLabel));
 	}
 
 	/**
 	 * Delete all nodes and relationships from this graph.
+	 * FIXME: Should be replaced by batchinserter code
```

Comment:
I think for reviewing this PR this method can be skipped, since @gfokkema explicitly mentioned this to be a temporary ugly fix.